tests/ui/UnreadIndicatorsTest.js

```diff
@@ -0,0 +1,395 @@
+/* eslint-disable @lwc/lwc/no-async-await */
+import React from 'react';
+import Onyx from 'react-native-onyx';
+import {Linking, AppState} from 'react-native';
+import {fireEvent, render, act} from '@testing-library/react-native';
+import lodashGet from 'lodash/get';
+import moment from 'moment';
+import App from '../../src/App';
+import CONST from '../../src/CONST';
+import ONYXKEYS from '../../src/ONYXKEYS';
+import waitForPromisesToResolve from '../utils/waitForPromisesToResolve';
+import * as TestHelper from '../utils/TestHelper';
+import appSetup from '../../src/setup';
+import fontWeightBold from '../../src/styles/fontWeight/bold';
+import * as AppActions from '../../src/libs/actions/App';
+import ReportHeaderViewBackButton from '../../src/pages/home/ReportHeaderViewBackButton';
+import ReportActionsView from '../../src/pages/home/report/ReportActionsView';
+import * as NumberUtils from '../../src/libs/NumberUtils';
+import LocalNotification from '../../src/libs/Notification/LocalNotification';
+import * as Report from '../../src/libs/actions/Report';
+
+beforeAll(() => {
+    global.fetch = TestHelper.getGlobalFetchMock();
+
+    // We need a bit more time for this test in some places
```

Comment:
Can you please enhance the comment to explain which tests? Why not just increase the timeout in the tests that are necessary?

Reply:
OK, after seeing how long the test is below, I can see why this might be necessary. What I would rather see though is have the test broken up into smaller testable chunks. It's basically a huge monolithic test right now and I think that splitting it up into smaller chunks will make it easier to reason about what the test is doing.

Reply:
Broke it up into separate independent tests and you were right! The extra time for the tests was able to be removed.

Reply:
Great! Mind removing the comment too? 😁

Reply:
Doh!